node/toolrunner.ts

```diff
@@ -584,27 +588,48 @@ export class ToolRunner extends events.EventEmitter {
                 this.pipeOutputToTool._getSpawnArgs(options),
                 this.pipeOutputToTool._getSpawnOptions(options));
 
+            let fileStream: fs.WriteStream = this.pipeOutputToFile ? fs.createWriteStream(this.pipeOutputToFile) : null;
+            if (fileStream) {
+                fileStream.write(this._getCommandString(options) + os.EOL);
```

Comment:
is it weird to pipe the command string? I would expect the file to only contain the output of the first tool.

Reply:
The command string helps set context for what this output was for. Otherwise user has to use the step number to match the output especially if there are multiple tasks of the same type that are producing these files. I initially did not write it but after I tried e2e it made more sense to write it.

Reply:
we cant do this. it may contain secrets and does not get run through the log scrubber

Reply:
Ah good point!